reana_client/utils.py

```diff
@@ -147,6 +148,12 @@ def _prepare_kwargs(reana_yaml):
                 )
             )
             _validate_reana_yaml(reana_yaml)
+        if environments:
+            logging.info(
+                "Validating environments in REANA specification file: "
+                "{filepath}".format(filepath=filepath)
+            )
+            _validate_environment_tags(reana_yaml)
```

Comment:
This function can be simply called `_validate_environments` as it will include image existence and uid/gid validation.